schema/codelists/partyRole.csv

```diff
@@ -1,21 +1,21 @@
 Code,Title,Description,Source,Deprecated
 buyer,Buyer,"The organization aiming to conclude a contract with a supplier or to use the goods, works or services resulting from the contract.",,
-procuringEntity,Procuring entity,"The organization managing the contracting process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",,
+procuringEntity,Procuring entity,"The organization managing the contracting (or planning) process. If an organization is both a buyer and a procuring entity (as can be the case in simple contracting processes), its roles should include 'buyer', but not 'procuringEntity'.",,
 supplier,Supplier,An organization with which a buyer or a procuring entity decided to conclude a contract.,,
 tenderer,Tenderer,An organization that submitted a bid.,,
 funder,Funder,The funder is an organization providing money or finance for this contracting process.,,1.2
-enquirer,Enquirer,An organization that has made an enquiry during the enquiry phase of a contracting process.,,
+enquirer,Enquirer,An organization that has made an enquiry during the enquiry phase of a contracting (or planning) process.,,
 payer,Payer,An organization making a payment from a transaction.,,
 payee,Payee,An organization in receipt of a payment from a transaction.,,
 reviewBody,Review body,An organization responsible for the review of this procurement process. This organization often has a role in any challenges made to the contract award.,,
 interestedParty,Interested party,"An organization that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",,
-contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting process.),,
+contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting (or planning) process.),,
```

Comment:
I'm not sure that a planning process would ever have a contractImplementationManager.

Reply:
Ok, removed in https://github.com/open-contracting/standard/pull/1513/commits/16d72e242f12f8642c19175d2c97bc3942d2dbb2

(My wording doesn't require there to be a contractImplementationManager (CIM) in the planning process - I was rather thinking that it's probably true/consistent to say that if there is a CIM in a contracting process, then the CIM might be different from the procuring entity that manages the contracting (_and/or_ planning) process. However, these are typically the same; instead of "manages" we might need to say "manages"/"managed" etc. - it doesn't matter. Removed :). )

Reply:
Yes, it was a true statement, but it kind of violates the Gricean maxim of quantity :)

> Do not make your contribution more informative than is required

https://psychology.fandom.com/wiki/Gricean_maxims#Maxim_of_Quality

Reply:
> I'm not sure ocid needs updating - the planning process is in the second sentence there?

Oh, yes, I was lazy and stopped reading after the first sentence :)